website/enroll/views.py

```diff
@@ -21,21 +22,25 @@ def gen_code() -> str:
 @require_http_methods(['POST'])
 def send(request):
     email = request.POST.get('email')
+    email=json.loads(email)
     log(email)
     obj = VerifyCodeModel.objects.filter(email=email).first()
     code = gen_code()
+    send_time=datetime.datetime.now()
     if obj is not None:
         obj.code = code
     else:
         try:
-            obj = VerifyCodeModel.objects.create(email=email, code=code)
+            obj = VerifyCodeModel.objects.create(email=email, code=code,send_time=send_time)
         except ValidationError:
             return JsonResponse(dict(detail="邮箱格式错误"), status=422)
     obj.save()
 
     log(code)
 
     err_msg = send_code(code, [email])
+    VerifyCodeModel.objects.filter(
+        send_time=datetime.datetime.now()-datetime.timedelta(minutes=10)).delete()#清除间隔发送时间10分钟的数据
```

Comment:
Make this execute before `obj` is queried from DB (i.e. `obj = VerifyCodeModel.objects.filter(email=email).first()`)